slurm-mlr/README.md

```diff
@@ -0,0 +1,19 @@
+# Disclaimer
+
+This example shows how to use `drake` to perform a benchmark analysis using two algorithms (KNN, SVM) across two datasets.
+
+The shown tuning ranges and settings of the cross-validation are not meant to make sense. 
```

Comment:
How about "The shown tuning ranges and settings of the cross-validation are scaled down for pedagogical purposes."?

Reply:
Sounds better :+1: